docs/external-node.md

```diff
@@ -266,13 +266,24 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
    ```
 
 2. The `antrea-agent.conf` file specifies agent configuration parameters. Copy
-   the [agent configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
-   to the VM and edit the `antrea-agent.conf` file to set `clientConnection`,
-   `antreaClientConnection` and `externalNodeNamespace` with the correct values.
-   Copy `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig` files to
+   the [agent configuration file](../build/yamls/externalnode/conf/antrea-agent.conf),
+   and `antrea-agent.antrea.kubeconfig` and `antrea-agent.kubeconfig` files to
    the VM, that were generated in the step 4 and step 5 of
    [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster).
 
+3. Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.sh)
+   as shown below and skip the remaining steps.
+
+    ```bash
+    ./install-vm.sh --ns vm-ns --bin ./antrea-agent --config ./antrea-agent.conf \
```

Comment:
We should use URL too, when we add binary downloading URLs.

Reply:
yes, this needs to be updated when we release.